munkipkg

```diff
@@ -639,6 +639,27 @@ def build(project_dir, options):
             if retcode:
                 raise BuildError("Could not remove existing %s" % outputname)
 
+        # remove .DS_Store files from the payload
```

Comment:
This isn't really necessary. From `man pkgbuild`:

```
--filter filter-expression
                 By default, --root will include the entire contents of the given root-path in the package pay-
                 load, except for any .svn or CVS directories, and any .DS_Store files. You can override these
                 default filters by specifying one or more --filter options. Each filter-expression is an
                 re_format(7) ``extended'' expression: any path in the root which matches any of the given expres-
                 sions will be excluded from the package payload. (Note that specifying even one --filter inhibits
                 the default filters, so you must respecify the default filters if you still want them to be
                 used.)
```

Note the "except for any .svn or CVS directories, and any .DS_Store files."

If you really think deleting these is needed, I'd like to know why you think so, and I'd prefer a Python implementation to shelling out to `find`, which then calls `rm`...